App.js

```diff
@@ -33,6 +35,7 @@ let deviceId =
 export default class App extends Component {
   componentDidMount() {
     store.dispatch(loadSettings(deviceId));
+    store.dispatch(fetchPosters(store.getState().settings.settings.language));
```

Comment:
Easier:
 mapDispatchToProps={
loadSettings,
fetchPosters
}
 componentDidMount{
    this.props.loadSettings
}

Reply:
I guess it will not work because App is not connected to store.

Reply:
you're right. good catch

Reply:
one more thing. I'm not sure but fetchPosters may obtain undefined because loadSettings works in async manner, so you need to wait for it to finish.

Reply:
it takes about 5 seconds to load whole slider screen, so delay for loadSettings is not too long.